src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -213,14 +212,6 @@ public void sanitizeForSaving() {
         this.courseId = Sanitizer.sanitizeForHtml(courseId);
         this.giverEmail = Sanitizer.sanitizeForHtml(giverEmail);
 
-        if (recipients != null) {
```

Comment:
Ah wait, maybe I should revert this part and assume `recipients` has been already sanitized and temporarily remove `escapeXml()` in the frontend.